src/styles/blocks/card.css

```diff
@@ -0,0 +1,93 @@
+.card {
+  display: inline-block;
+  width: 200px;
+  padding: 32px 16px 16px 16px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  --main-card-text-color: #060B35;
+  --second-card-text-color: #616070;
+}
+
+.card__img {
+  display: block;
+  margin: 0 3px 40px 3px; 
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  width: 100%;
+  margin-bottom: 4px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: var(--main-color);
+}
+
+.card__product-code {
+  margin-bottom: 16px;
+  text-align: left;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--additional-color);
+}
+
+.card__stars-reviews {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.card__reviews-text {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+}
+
+.card__price-text {
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--additional-color);
+}
+
+.card__price-value {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--main-color);
+}
+
+.card__button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-decoration: none;
+  box-sizing: border-box;
```

Comment:
according to previous mentors comments - move box-sizing: border-box; to * selector